coalib/bears/Bear.py

```diff
@@ -266,7 +266,9 @@ def __json__(cls):
         """
         Override JSON export of ``Bear`` object.
         """
-        _dict = get_public_members(cls)
+        # json cannot serialize properties, so drop them
```

Comment:
why not make `__json__` a method and instead of `key: value` use `key: getattr(self, key)`?